### PR TITLE
Update Environment for Pagination

### PR DESCRIPTION
**Pull request by Suyash-d**
**Is your feature request related to a problem? Please describe.**

Create Pagination request of Environment for Open API documentation.

### CODE REVIEW

File: src/paths/environments/fetchAll.yaml [hunk 1]
```diff
@@ -4,6 +4,9 @@ tags:
 operationId: environmentsFetchAll
 security:
   - bearerAuth: ["ENVIRONMENTS_READ"]
+parameters:
+  - $ref: "../../parameters/query/pagination/Page.yaml"
+  - $ref: "../../parameters/query/pagination/Size.yaml"
 responses:
   '200':
     description: Expected response to a valid request
```
Comment (BerrevoetsRobbe) [inline src/paths/environments/fetchAll.yaml:12]: "../../schemas/environments/EmbeddedEnvironmentDto.yaml"
should be modified as well. Did you visually confirm the layout?

**Review by BerrevoetsRobbe — CHANGES_REQUESTED** [review]

File: src/paths/environments/fetchAll.yaml (hunk 1 above)
Comment (BerrevoetsRobbe) [reply src/paths/environments/fetchAll.yaml:12]: response does not contain pagination information. have a look at src/paths/scripts/fetchAll.yaml for inspiration

**Review by BerrevoetsRobbe — CHANGES_REQUESTED** [review]

File: src/paths/environments/fetchAll.yaml (hunk 1 above)
Comment (BerrevoetsRobbe) [reply src/paths/environments/fetchAll.yaml:12]: src/schemas/scripts/PagedScriptDto.yaml

**Review by BerrevoetsRobbe — APPROVED** [review]

**Comment by BerrevoetsRobbe** [comment]
linked to metadew/iesi#352

**Comment by Suyash-d** [comment]
**This PR to be merged in this branch before merged to develop [Unpaginated Get Endpoint for Environment #24]**(https://github.com/metadew/iesi-api-documentation/pull/24)

**Comment by BerrevoetsRobbe** [comment]
don't forget to build and push the build openapi.yaml

**Comment by Suyash-d** [comment]
@BerrevoetsRobbe Please merge to develop, [fetch all environments unpaginated](https://github.com/metadew/iesi/pull/368) merged to this branch.

**Review by BerrevoetsRobbe — APPROVED** [review]

**Pull request merged**